README.md

```diff
@@ -1,5 +1,9 @@
+[![Travis CI](https://travis-ci.org/anthonydresser/vscode-mssql.svg?branch=dev)](https://travis-ci.org/anthonydresser/vscode-mssql)
```

Comment:
Once you enable travis, coveralls, and appveyor, you just need to update the reference.